dissect/target/loaders/velociraptor.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from dissect.target.loaders.dir import DirLoader, find_dirs, map_dirs
+
+if TYPE_CHECKING:
+    from dissect.target import Target
+
+ROOT_FILESYSTEM = "uploads"
+
+
+def find_os_directory(path: Path):
+    # As of Velociraptor version 0.6.7 the structure of the Velociraptor Offline Collector varies by operating system
+    # Generic.Collectors.File (Unix, OS-X) root filesystem is 'uploads/'
+    # Generic.Collectors.File (Windows) and Windows.KapeFiles.Targets (Windows) root filesystem is
+    # 'uploads/<file-accessor>/<drive-name>/'
+    os_type, dirs = find_dirs(path.joinpath(ROOT_FILESYSTEM))
+    if os_type in ["linux", "osx"]:
+        return dirs[0]
+
+    # This suppports usage of the ntfs accessor 'uploads/mft/%5C%5C.%5CC%3A' not the accessors lazy_ntfs or auto
+    if not os_type:
+        # Filter out files that start with '.'
+        windows_path = [p for p in path.joinpath(ROOT_FILESYSTEM + "/mft").iterdir() if not p.name.startswith(".")][0]
```

Comment:
```suggestion
        windows_path = [p for p in path.joinpath(ROOT_FILESYSTEM).joinpath("/mft").iterdir() if not p.name.startswith(".")][0]
```
Better to individually join the `mft` path.

Reply:
Also, what if the `mft` directory doesn't exist?

Reply:
The function should return `None`